.env

```diff
@@ -87,4 +87,15 @@ KC_DB_OPTS="--db postgres --db-url jdbc:postgresql://kc-ssi-instance-1.clh0lvey1
 # For AWS: Remember to provide cert and key pem files ($WORK_DIR/../env/.env)
 KC_START="start --hostname-strict=false --https-port=$KEYCLOAK_HTTPS_PORT --https-certificate-file=$KC_SERVER_CERT --https-certificate-key-file=$KC_SERVER_KEY"
 # For local env
-# KC_START=start-dev
\ No newline at end of file
+# KC_START=start-dev
+
+# Keycloak config CLI
+REPO_URL="https://github.com/adorsys/keycloak-config-cli.git"
+KC_CLI_DIR=$(pwd)/config
+JAR_FILE=keycloak-config-cli.jar
+KEYCLOAK_URL=https://kc-ssi.solutions.adorsys.com/
+KEYCLOAK_USER=$KEYCLOAK_ADMIN
+KEYCLOAK_PASSWORD=$KEYCLOAK_ADMIN_PASSWORD
+IMPORT_PATH=$KC_CLI_DIR/realm.json
+PROJECT_DIR=$KC_CLI_DIR/keycloak-config-cli
```

Comment:
There are lots of env variables... Can you think of more specific names for JAR_FILE, IMPORT_PATH, and PROJECT_DIR?